### ISSUE TITLE
Date of the compile

### ISSUE THREAD

**Issue by jglerner** [LidorBaum/BiquB1MarlinFirmware#1]
I’d suggest you consider adding the date of the compile of the firmware.Something like 20211119 string or so.

**Comment by LidorBaum** [comment]
Thank you for your comment. 
I am planning to compile new firmwares with the lastest Marlin 2.0.9.x . I will add then the date of compiling.

**Comment by pimpmytank** [comment]
This is great. Just got my 2nd BIQU B1. Had no issue compiling the firmware for the first printer but didn't realize this one had a different board. Do you have a version where the Z OFFSET WIZARD is enabled?

**Comment by LidorBaum** [comment]
@pimpmytank Z offset can be adjusted via the screen. or maybe I did not understand your question? can you please explain?

**Comment by pimpmytank** [comment]
Agreed, I've not had great outcomes setting it through the screen. In Marlin, after homing, you can set it using the "Z Probe Wizard" (screen shot attached) if it's enabled in the "Configuration_adv.h" file and compiled with the firmware. I've tried to configure firmware for SKR2 and enable this but get errors
![probeoffsetmarlin](https://user-images.githubusercontent.com/94937148/143250715-506939e1-332b-4e62-afd0-c7c4269daeee.png)
s.
